main.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+# *_* coding: utf-8 *_*
+"""This module deploys discord bot using discord.py library."""
+
+import time
+import os
+from typing import Optional
+import discord
+from discord import app_commands
+from discord.ext import commands
+from dotenv import load_dotenv
+from config_creator import (
+    check_for_config_file,
+    create_config_ram,
+    read_from_config,
+    save_values_from_ram_to_memory,
+)
+from embed_creator import EmbedCreator, HelpMenu, auto_update
```

Comment:
Z tego co widzę to w tym pliku są głównie widoki oraz różne elementy interfejsu.
Może lepszą nazwą byłoby coś co sugeruje, że właśnie stąd bierzemy widoki i elementy UI?
Może `bot_gui`?